installing_on_vmware_vsphere/_topics/installation.md

```diff
@@ -33,7 +33,7 @@ has the following requirements:
 **Note:**
 
 These are the procedural steps as of the time of writing. For more
-information, consult the "Deploying an OVF or OVA Template" topic in the [VMware documentation](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-17BEDA21-43F6-41F4-8FB2-E01D275FE9B4.html).
+information, consult the `Deploying an OVF or OVA Template' topic in the [VMware documentation](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-17BEDA21-43F6-41F4-8FB2-E01D275FE9B4.html).
```

Comment:
The link looks identical but the PR description says the link changed, so I'm curious if that was left out accidentally?

Otherwise it looks like the quoting changed, but it from from a pair of double quotes to a backtick/single-quote pair.  It should probably stay as a pair of double quotes.